Hw03-JoshWakin.py

```diff
@@ -0,0 +1,86 @@
+#################################################################################
+# Author: Josh Wakin
+# Username:thealphagurlux
+#
+# Assignment: HW03
+# Purpose: To draw a circle, fill it, then draw a * over it.
+# Google Doc Link:https://docs.google.com/document/d/1638XgwboqPy_1V2K1NU523jD-TeApx-yEHIs-DOrhQ8/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#################################################################################
+
+import turtle
+
+
+t1=turtle.Turtle() #turtle to draw with
+t1=turtle.penup()  #pull the turtles pen up because we are going to offset it
+t1=turtle.pensize(30)
+wn=turtle.Screen()
+wn.bgcolor("Blue")
```

Comment:
No code outside of functions/main!

This should be the first five lines of `main()`